3.sql

```diff
@@ -0,0 +1 @@
+select * from profitshop.products where createdAt LIKE '2021-03-01%';
```

Comment:
A date comparison is expected here. See, string operations are way slower. Moreover, the actual data is stored as DATE, and you invoke a kind of coercion for this constraint, which builds up to the execution time